_dropin-templates/dropin-functions.mdx

```diff
@@ -1,33 +1,58 @@
 ---
 title: DROPIN_NAME functions
-description: Learn about the API functions provided by the DROPIN_NAME drop-in.
+description: Learn about the API functions provided by the DROPIN_NAME drop-in component.
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 2
 ---
 
+import OptionsTable from '@components/OptionsTable.astro';
 import Aside from '@components/Aside.astro';
+import CodeInclude from '@components/CodeInclude.astro';
 
-This topic provides the details and instructions you need to use the functions provided by the DROPIN_NAME drop-in.
+The DROPIN_NAME drop-in component provides API functions that allow developers to retrieve and display detailed information dynamically.
```

Comment:
Drop-in component API functions allow developers to integrate their own internal logic, enabling the creation of custom behaviors and functionality.